### PR TITLE
feat(docs): Add MySQL Logs doc

### PR DESCRIPTION
**Pull request by crazyuploader**
## Pull Request

---

### 📄 Summary
This PR adds MySQL logs doc

#### Issues closed by this PR
Closes https://github.com/SigNoz/growth-pod/issues/1056

Doc Links:

- [MySQL - Logs Doc](https://staging.signoz.io/docs/integrations/opentelemetry-mysql/)
- [MySQL - Metrics Doc](https://staging.signoz.io/docs/metrics-management/mysql-metrics/#step-4-collect-query-samples-and-top-queries)

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

---

- Tests added/updated:
- Manual verification:
- Edge cases covered:

---

- Blast radius:
- Potential regressions:
- Rollback plan:

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Feature / Bug Fix / Maintenance |
| Description | User-facing summary |

---

### 📋 Checklist
- [ ] Tests added or explicitly not required
- [ ] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

---

## 👀 Notes for Reviewers

---

### CODE REVIEW

File: data/docs/metrics-management/mysql-metrics.mdx [hunk 1]
```diff
@@ -56,6 +59,37 @@ service:
 
 Now, run the OpenTelemetry collector and you should see the metrics coming in SigNoz. 
 
+<details>
+<ToggleHeading>
+
```
Comment (revmag) [inline data/docs/metrics-management/mysql-metrics.mdx:51]: This shouldnt be step 4, you can just add this in Step 2, with a toggle, like want to collect metrics from logs too, or something like this

File: data/docs/integrations/opentelemetry-mysql.mdx [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+The error log records startup, shutdown, and runtime problems. Configure a file destination that the Collector can read:
+
+```ini:my.cnf
+[mysqld]
+log_error = /var/log/mysql/error.log
+```
+
+Default location varies by distribution:
+
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-mysql.mdx:45]: Maybe make this in a toggle?

File: data/docs/integrations/opentelemetry-mysql.mdx [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+slow_query_log = 1
+slow_query_log_file = /var/log/mysql/mysql-slow.log
+long_query_time = 2
+# Optional: log statements that run without an index
+log_queries_not_using_indexes = 1
+```
+
+The path above follows the same convention as the error log. MySQL's actual default for `slow_query_log_file` is `<hostname>-slow.log` in the data directory (typically `/var/lib/mysql/`). Use any absolute path the Collector can read.
+
+**Configuration parameters:**
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-mysql.mdx]: This can go in toggle

File: data/docs/integrations/opentelemetry-mysql.mdx [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+
+- `slow_query_log`: set to `1` to enable.
+- `slow_query_log_file`: absolute path to the log file the Collector will tail.
+- `long_query_time`: threshold in seconds (default `10`, supports microsecond precision such as `0.5`).
+- `log_queries_not_using_indexes`: optional, logs full-scan queries regardless of duration.
+
+A slow query log entry spans multiple lines and looks like this:
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-mysql.mdx]: Toggle again

File: data/docs/integrations/opentelemetry-mysql.mdx [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+<Admonition type="info" title="Using self-hosted SigNoz?" defaultCollapsed="true">
+Most steps are identical. Update the exporter endpoint and remove the ingestion-key header as shown in [Cloud to Self-Hosted](https://signoz.io/docs/ingestion/cloud-vs-self-hosted/#cloud-to-self-hosted).
+</Admonition>
+
+## Prerequisites
+
+- A MySQL 8.0+ server that you can access and restart
+- An [OpenTelemetry Collector](https://signoz.io/docs/opentelemetry-collection-agents/vm/install/) v0.144.0+ on the MySQL host, running as a user with read access to the log files
+
+## Enable MySQL Logs
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-mysql.mdx]: Time to first data send is not clear in this case, if there are things that can be direclty demonstrated, we should do that.
Think about what the user might know and how we can easily send data. It looks complex this doc

**Comment by revmag** [comment]
Resolve merge conflict

**Review by revmag — APPROVED** [review]
